src/mon/HealthMonitor.h

```diff
@@ -52,6 +52,7 @@ class HealthMonitor : public PaxosService
     bool want_detail,
     ceph::Formatter *f,
     std::string *plain,
+    bool truncate_detail = false,
```

Comment:
@neha-ojha It's because of optional argument ordering constraint. If we skip passing argument truncate_detail where we are passing values for sep1 and sep2 then compiler complains about converting ‘std::nullptr_t’ to 'bool' here. I think, it's better to pass optional argument truncate_detail at the end of argument list so that we donot have to change the get_health_status calls everywhere.

Reply:
makes sense